engine/pkg/externalresource/model/config.go

```diff
@@ -31,35 +32,73 @@ var DefaultConfig = Config{
 		Bucket: "",
 		Prefix: "",
 	},
+	GCS: GCSConfig{
+		Bucket: "",
+		Prefix: "",
+	},
 }
 
 // Config defines configurations for an external storage resource
 type Config struct {
```

Comment:
Perhaps it is better to configure external storage through URI, which are more compatible with other storage types (such as azure).

Reply:
Yes, it's more easier to support other storage, but I find it hard to tell what kind of parameters the `Storage` offers because `URI` is a simple string.